packages/apollo-codegen-swift/src/__tests__/__snapshots__/codeGeneration.ts.snap

```diff
@@ -2,8 +2,14 @@
 
 exports[`Swift code generation #classDeclarationForOperation() should generate a class declaration for a mutation with variables 1`] = `
 "public final class CreateReviewMutation: GraphQLMutation {
+  /// mutation CreateReview($episode: Episode) {
+  ///   createReview(episode: $episode, review: {stars: 5, commentary: \\"Wow!\\"}) {
+  ///     stars
+  ///     commentary
+  ///   }
+  /// }
   public let operationDefinition =
-    \\"mutation CreateReview($episode: Episode) {\\\\n  createReview(episode: $episode, review: {stars: 5, commentary: \\\\\\"Wow!\\\\\\"}) {\\\\n    stars\\\\n    commentary\\\\n  }\\\\n}\\"
+    \\"mutation CreateReview($episode: Episode) {\\\\n createReview(episode: $episode, review: {stars: 5, commentary: \\\\\\"Wow!\\\\\\"}) {\\\\n stars\\\\n commentary\\\\n }\\\\n }\\"
```

Comment:
It looks like you're trimming adjacent spaces but doing nothing about the embedded newlines.

Reply:
Ah I think I missed something on the request there. This should now be updated.